source/code/data_types.md

```diff
@@ -351,3 +351,30 @@ The `data-wb-answer` value must match the value of the question to which it corr
 
 <aside class="tip">We have established a shorthand-plus-script workflow to make adding workbook markup easier, see [expand-ws.js](https://github.com/bhdirect-ebooks/single-scripts/tree/master/expand-wb) for information.
 </aside>
+
+### Links to CROSS Books
+
+Two different syntaxes work for creating links to other CROSS books:
+
+```
+<a data-link='{"pub":"0","book":"19768","bookmark":"page3"}'>
```

Comment:
Maybe we should add `...</a>` here and in the other code example, just to clarify the tag needs to be completed. Everybody *should* know that, but it's good to be as clear as possible?